app/jobs/resolution_proofing_job.rb

```diff
@@ -131,71 +131,57 @@ def proof_lexisnexis_ddp_with_threatmetrix_if_needed(
 
   # @return [CallbackLogData]
   def proof_lexisnexis_then_aamva(timer:, applicant_pii:, should_proof_state_id:)
-    proofer_result = timer.time('resolution') do
+    resolution_result = timer.time('resolution') do
       resolution_proofer.proof(applicant_pii)
     end
 
-    result = proofer_result.to_h
-    resolution_success = proofer_result.success?
-
-    result[:transaction_id] = proofer_result.transaction_id
-    result[:reference] = proofer_result.reference
-
-    exception = proofer_result.exception.inspect if proofer_result.exception
-    result[:timed_out] = proofer_result.timed_out?
-    result[:exception] = exception
+    state_id_result = Proofing::Aamva::UnsupportedJurisdictionResult.new
+    if should_proof_state_id && resolution_result.success?
+      timer.time('state_id') do
+        state_id_result = state_id_proofer.proof(applicant_pii)
+      end
+    end
 
-    result[:context] = {
-      should_proof_state_id: should_proof_state_id,
-      stages: {
-        resolution: {
-          client: resolution_proofer.class.vendor_name,
-          errors: proofer_result.errors,
-          exception: exception,
-          success: proofer_result.success?,
-          timed_out: proofer_result.timed_out?,
-          transaction_id: proofer_result.transaction_id,
-          reference: proofer_result.reference,
+    result = {
+      success: resolution_result.success?,
+      errors: resolution_result.errors.merge(state_id_result.errors),
+      exception: resolution_result.exception || state_id_result.exception,
+      timed_out: resolution_result.timed_out? || state_id_result.timed_out?,
+      context: {
+        should_proof_state_id: should_proof_state_id,
+        stages: {
+          resolution: {
+            client: resolution_proofer.class.to_s,
```

Comment:
Here I think we it may be worth doing something like this:

```ruby
stages: {
  resolution: resolution_result.to_h,
  state_id: state_id_result.to_h,
}
```

Reply:
Doing this means we will change `client` to `vendor_name` and the output there will be closer to what we were seeing before when the value was the vendor name value.